components/statusBar.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+  display: flex;
+`;
+
+const Bar = styled.div`
+  height: 20px;
+  width: ${({ width }) => width}px;
```

Comment:
Would change to 100% width so that the parent container is the one owning the size of this component!

Reply:
its been done director

Reply:
Thank you sir 🚀 merge whenever!